### PR TITLE
feat: new editor tip tap

### PR DESCRIPTION
**Pull request by pradel**
First version of the editor. A lot missing but we already have a solid base with the basic features.
I created a /tiptap-editor page that can be accessed at any time to try it out.
Demo https://sigle-m4hkekfid-leopradel.vercel.app/tiptap-editor

### CODE REVIEW

File: sigle/src/modules/editor/TipTapEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      SlashCommands.configure({
+        commands: slashCommands,
+        component: SlashCommandsList,
+      }),
+    ],
+    content: '<p>Hello World! 🌎️</p>',
+  });
+
+  const addImageToEditor = (files: FileList | null) => {
+    if (!files) {
```
Comment (pradel) [inline sigle/src/modules/editor/TipTapEditor.tsx:143]: will be done in a separate pr as it's a big task

**Review by Akirtovskis — APPROVED** [review]

**Review by gregogun — APPROVED** [review]
